include/nbl/ext/RadixSort/RadixSort.h

```diff
@@ -88,7 +100,10 @@ class RadixSort final : public core::IReferenceCounted
 		driver->updateDescriptorSets(descriptors.size(), writes, 0u, nullptr);
 	}
 
-	const uint32_t m_wg_size;
+	static void exclusiveSumScan(video::IVideoDriver* driver, core::smart_refctd_ptr<video::IGPUBuffer> in_gpu,
+		video::IGPUDescriptorSet* ds_upsweep, video::IGPUComputePipeline* upsweep_pipeline, video::IGPUDescriptorSet* ds_downsweep,
+		video::IGPUComputePipeline* downsweep_pipeline, ScanClass::Parameters_t* push_constants, ScanClass::DispatchInfo_t* dispatch_info,
+		const uint32_t total_pass_count, const uint32_t upsweep_pass_count);
```

Comment:
seems like either bad a name or being in the wrong file

Reply:
not quite, look at the corresponding usage here https://github.com/Devsh-Graphics-Programming/Nabla/blob/59f3cc4237b4f6043c9ab79983132aca31a34e0c/examples_tests/51.RadixSort/main.cpp#L141

Reply:
do you want me to remove the function and inline the code?

Reply:
whatever, but I was complaining about the fact that a function that is 100% to do with scanning, was in the radix sort header/class